proto/streamlit/proto/NewSession.proto

```diff
@@ -167,7 +167,9 @@ message FontSizes {
 message UserInfo {
   string installation_id = 1;
   string installation_id_v3 = 5;
-  string email = 2;
+
+  // DEPRECATED:
+  // string email = 2;
```

Comment:
I'm not 100% sure about the latest state of protobuf compatibility. Can I actually remove this property here, or should I keep it in for compatibility reasons even though it won't get used anymore? @vdonato @tconkling

Reply:
it's fine to remove it - our current pattern is to replace the removed field `reserved 2;` so that if someone tries to reuse the ID it will actually throw a protobuf compile error.

Reply:
(So we should do that here)